src/Orleans.Core/Shims/RuntimeStatisticsGroup.cs

```diff
@@ -1,34 +1,22 @@
-﻿using System;
+using System;
 using Microsoft.Extensions.Logging;
+using Orleans.Statistics;
 
 namespace Orleans.Runtime
 {
-    internal class RuntimeStatisticsGroup : IDisposable
+    internal class DummyHostEnvironmentStatistics : IHostEnvironmentStatistics
```

Comment:
naming Dummy is not clear of what it is doing. Maybe `NoOpHostEnvironmentStatistics`?